### ISSUE TITLE
question about color of

### ISSUE THREAD

**Issue by MengHao666** [facebookresearch/InterHand2.6M#78]
there are a lot of colors of hands in your dataset, some of them are purely gray image, and other of them are rgb image with colors

**Comment by mks0601** [comment]
What is your question?

**Comment by rohitdavas** [comment]
Hi, The interhand dataset has mostly yellowish images or whitish hands. The hands do not look quite real as in natural light sense. After analysis I found that hand images color in R and G space and B space is quite close. And hence we can use color stretching to come to real looking images. Is there any better way ? 

(64, 64, 4)

before. 

R values. (min, max) = (0.0, 146.0)
G values. (min, max) = (0.0, 139.0)
B values. (min, max) = (0.0, 133.0)

after
stretch. R values. (min, max) = (0.0, 225.2)
stretch. G values. (min, max) = (0.0, 175.10000000000002)
stretch. B values. (min, max) = (0.0, 156.4)

![Figure_3](https://user-images.githubusercontent.com/52150408/133427865-94221deb-aa18-49bc-a854-d1c2d03f073a.png)

**Comment by mks0601** [comment]
@rohitdavas That's true. The unrealistic color of images is due to the capture system's light setting and cameras' color calibration.

**Comment by rohitdavas** [comment]
So is there any better way to make them look realistic ? Otherwise the dataset will not be useful in real settings.

**Comment by mks0601** [comment]
Actually, the different color spaces are applicable to all motion capture datasets, such as Human3.6M. The motion capture datasets provide accurate GT 3D pose, but have limited or different image appearance. The in-the-wild datasets, such as MSCOCO, have GT 2D pose (no 3D pose), but have diverse and realistic image appearance. There are trade-offs between two types of dataset and you should jointly train a 3D hand pose estimator on both types of dataset to get benefit of two types of datsets.

**Comment by mks0601** [comment]
Simply changing colors of hand skins cannot solve the image appearance domain gap between this dataset and real images because of backgrounds, motion blurs of real images, and so on.

**Comment by mks0601** [comment]
I'm working on my new project for 3D interacting hand pose estimation in the wild. Please stay tuned.

**Comment by rohitdavas** [comment]
Thanks. would love to hear from your hand pose estimation in the wild project.

**Comment by amundra15** [comment]
Are the color calibration parameters of the cameras available?

**Comment by mks0601** [comment]
Sorry I do not have :(

**Comment by amundra15** [comment]
I see. And the radial distortion parameters?

**Comment by mks0601** [comment]
That neither :(
